### DISCUSSION TITLE
Add Options Support on Embedded Profile and Future TurboWarp/Forkphorus Player add-on

### DISCUSSION THREAD

**Discussion by SamuelTheTechnoKid** [ScratchAddons/ScratchAddons#3159] [Discussion]
Hello, Scratch Addons community! I want you to do something. Please, please, please add my suggestion on the title. Also, when is the next update coming?

**Comment by SamuelTheTechnoKid** [comment]
...

**Reply by CST1229** [reply]
What?

**Reply by SamuelTheTechnoKid** [reply]
Why did you move it to discussions?

**Reply by CST1229** [reply]
I didn't move it to discussions. Someone else did.

**Reply by WorldLanguages** [reply]
Issues should be used for specific things
